src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -504,7 +511,7 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
                 try
                 {
                     await InvokeAsync(invoker, parameterHelper, timeoutTokenSource, functionCancellationTokenSource,
-                    throwOnTimeout, timerInterval, instance);
+                    throwOnTimeout, timerInterval, instance, logger, jobHost, config);
```

Comment:
>, logger, jobHost, config [](start = 59, length = 25)

Let's leave the InvokeAsync() signature untouched and instead handled Filters here. That will also minimize test churn in FunctionExecutorTests.